explorer/scan/local_scanner.go

```diff
@@ -243,30 +246,39 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		}
 	}()
 
+	spaceMrn := ""
+	var services *explorer.Services
+	if upstream != nil && upstream.ApiEndpoint != "" && !upstream.Incognito {
+		client, err := upstream.InitClient()
+		if err != nil {
+			return nil, err
+		}
+		spaceMrn = client.SpaceMrn
+
+		services, err = explorer.NewRemoteServices(client.ApiEndpoint, client.Plugins, client.HttpClient)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	assetBatches := slicesx.Batch(discoveredAssets.Assets, 100)
 	for i := range assetBatches {
+		log.Info().Msg("synchronize assets")
```

Comment:
This will show the message also in cases where we do not have an upstream config.

Reply:
This should be somewhere between L270 and L304.

Reply:
fixed